### ISSUE TITLE
Track expansion state of tree views for tabs that are out of view

### ISSUE THREAD

**Issue by fschwiet** [usebruno/bruno#3431]
### I have checked the following:

- [X] I've searched existing issues and found nothing related to my issue.

### Describe the feature you want to add

When I change tabs Bruno should recall the expansion state of the view within a tab when I return to that tab.

Let me describe an example to help make that clear. As part of debugging my app, I am trying to manually compare specific parts of different responses coming from my app. My collection has two requests defined. I run one, and then use the expand/collapse to find the part I want to compare. I then run the other request and then use the expand/collapse to find the part of the response I want to compare. However now when I switch back to view the results of the first request the expand/collapse state has been reset, so I don't have the part I wanted to compare ready anymore.

### Mockups or Images of the feature

![remember expansion state](https://github.com/user-attachments/assets/bf5b57c2-5f2f-48b3-8df1-a19979624913)

**Comment by tapioko** [comment]
Duplicate of https://github.com/usebruno/bruno/issues/874 ?

**Comment by fschwiet** [comment]
Yes, sorry my search must have missed it because I searched for expansion and not expanse.
